tests/MIRI_ecfs/S4_MIRI.ecf

```diff
@@ -25,6 +25,15 @@ maxiters		5		# The number of iterations of sigma clipping that should be perform
 boundary        'fill'  # Use 'fill' to extend the boundary values by the median of all data points (recommended), 'wrap' to use a periodic boundary, or 'extend' to use the first/last data points
 fill_value		mask	# Either the string 'mask' to mask the outlier values (recommended), 'boxcar' to replace data with the mean from the box-car filter, or a constant float-type fill value.
 
+#Limb-darkening parameters needed to compute exotic-ld
+compute_ld      False
+inst_filter     LRS #filter of JWST/HST instrument, supported list see https://exotic-ld.readthedocs.io/en/latest/views/supported_instruments.html 
+metallicity     0.1 #metallicity of the star 
+teff            6000 #effective temperature of the star in K
+logg            4.0 #surface gravity in log g
+exotic_ld_direc /exotic-ld_data/ #directory for ancillary files for exotic-ld
```

Comment:
Is this folder with respect to some other folder? Surely the files aren't being saved in the root directory of your computer?